src/uct/api/uct.h

```diff
@@ -1168,6 +1217,37 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief Initiate a client-server connection to a remote peer.
+ *
+ * This routine will create an endpoint for a connection to the remote peer.
+ * The connection needs to be established by connecting to the
```

Comment:
This routine will create an endpoint for a connection to the remote peer, specified by its socket address.